src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,18 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos:Todo[]
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
+    {todos?.map(todoArrayElement => (
+      <TodoInfo
+        key={todoArrayElement.id}
+        todo={todoArrayElement}
+      />
+    ))}
```

Comment:
Use more descriptive names and try not to use types names.

```suggestion
    {todos?.map(todoInfo => (
      <TodoInfo
        key={todoInfo.id}
        todo={todoInfo}
      />
    ))}
```